ansible/roles/nginx/templates/nginx.conf.j2

```diff
@@ -0,0 +1,29 @@
+# {{ ansible_managed }}
+user {{ nginx_user }};
+worker_processes  {{ ansible_processor_count }};
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  {{ nginx_max_clients }};
+}
+
+
+http {
+
+        include /etc/nginx/mime.types;
+        default_type application/octet-stream;
+
+        access_log {{ log_home }}/{{ nginx_access_log_name }};
+        error_log {{ log_home }}/{{ nginx_error_log_name }};
```

Comment:
may want to add `warn` just in case? at least at the beginning. i don't actually know if we use warn in our code though